tests/onnxruntime/test_modeling.py

```diff
@@ -846,12 +846,9 @@ def test_save_load_decoder_model_with_external_data(self, use_cache: bool):
 
             # verify external data is exported
             folder_contents = os.listdir(tmpdirname)
-            self.assertTrue(ONNX_DECODER_NAME in folder_contents)
-            self.assertTrue(ONNX_DECODER_NAME + "_data" in folder_contents)
-
-            if use_cache:
-                self.assertTrue(ONNX_DECODER_WITH_PAST_NAME in folder_contents)
-                self.assertTrue(ONNX_DECODER_WITH_PAST_NAME + "_data" in folder_contents)
+            self.assertTrue(ONNX_WEIGHTS_NAME in folder_contents)
+            # self.assertTrue(ONNX_WEIGHTS_NAME + "_data" in folder_contents)
```

Comment:
will remove this test entirely if we keep post processing default for decoder models wtih  [`no_post_process=False`](https://github.com/huggingface/optimum/blob/refactorization-decoder-ort/optimum/onnxruntime/modeling_decoder.py#L593)
https://github.com/huggingface/optimum/blob/c98cb87e3b7151627ddc562341ffa46e39b1def0/optimum/exporters/onnx/base.py#L513

Reply:
added a slow tets instead in https://github.com/huggingface/optimum/pull/1257/commits/5406f95b52756a831e2ef24d609c9a42628164e8 after dicussion with @fxmarty